stdlib/Artifacts/test/runtests.jl

```diff
@@ -91,6 +91,9 @@ end
         HelloWorldC_exe_path = joinpath(HelloWorldC_dir, "bin", "hello_world$(exeext)")
         @test isfile(HelloWorldC_exe_path)
 
+        HelloWorldC_dir_explicit_artifact = eval(Expr(:macrocall, Symbol("@artifact_str"), nothing, "HelloWorldC", nothing, joinpath(@__DIR__, "Artifacts.toml")))
```

Comment:
`:macrocall` is generally not something that should be constructed by hand. `eval(:(@artifact_str "HelloWorldC" joinpath(@__DIR__, "Artifacts.toml"))` would be a better way to write that

Reply:
Ok, I just copied https://github.com/JuliaPackaging/JLLWrappers.jl/blob/79226fad1b220ba7a3516a66e76922eb84f293ae/src/wrapper_generators.jl#L17.